lib/rack/tracker/bing/template/bing.erb

```diff
@@ -0,0 +1,23 @@
+<% if events.any? %>
+<script src="//bat.bing.com/bat.js" async=""></script>
+<script>
+  <% events.each do |event| %>
+  function reportCustomEvent() {
```

Comment:
does this really work? 
it looks to me like for each event the function is overwritten? 
should the `events.each` be inside the function body (`function reportCustomEvent() {` ?

Reply:
I'm no expert either and the bing documentation is pretty limited...

Been following this page (which seems to be the official doc!! - they actually recommend 'View Source' to see the code):
http://bingadsuet.azurewebsites.net/UETDirectOnSite_ReportCustomEvents.html

None of the examples talk about having multiple events in one call but you're right it would probably make more sense to have multiple `window.uetq = ...` commands inside the `reportCustomEvent` function. Will amend that

Reply:
done

Reply:
ok, somehow I think we should even not have the function at all: https://help.bingads.microsoft.com/#apex/3/en/56684/0
I read it in the way that one can trigger such event for example on a click by the user. But we want to add the events once the page is loaded. 

maybe you can really test it from your branch and see if the events end up in bing? to be sure, then we merge?

Reply:
yes sounds like a plan. We'll deploy this today so hopefully will have some data soon..

Also hoping that someone else using this library might have more bing knowledge and chip in....

Reply:
You were right, the `function` bit is only required if want to tie the triggering of the conversion to a specific action, like an `onClick` on a button press.

I've amended my PR to call the event actions directly (the UET tag helper chrome extension detects the event being fired) but I'm still waiting to see if it correctly registers in the Bing dashboard (which can take up to 24hrs after an event occurred).

Will update here once I've confirmed it's working.

Reply:
🎉 It works!

<img width="1067" alt="bing" src="https://user-images.githubusercontent.com/895133/54616502-9b528f00-4a57-11e9-9bb8-8e7538080142.png">

Reply:
ok, great! then let's merge, or do you see anything missing/

Reply:
Ready to merge